### ISSUE TITLE
Just have a notification that says there are new assignments,  rather than the specific assignment name. 

### ISSUE THREAD

**Issue by thequbit** [hhroc/yellr-android#165]
Additionally, we should keep track of the assignments that we display this for (for all time),  and not re-did play them.   This will prevent the user being bombarded with notifications. 

@reustonium thoughts?

**Comment by reustonium** [comment]
I agree, to me this is a core difference between pinging the server for 'whats new', and true push notifications that happen in real time.

A slightly different approach is to 'stack' the notification like gmail (and others) doez.  https://developer.android.com/training/wearables/notifications/stacks.html

**Comment by thequbit** [comment]
I really do like the stack approach that gmail uses.  But yea, I agree ... it's more of a 'there is new stuff!' rather than a push.

**Comment by thequbit** [comment]
@dkd903 How is this done in the iOS version?  I seem to remember this is implemented as "you have n new notifications", right?
